modules/network-observability-architecture.adoc

```diff
@@ -0,0 +1,17 @@
+//Module included in the following assemblies:
+//
+// network_observability/understanding-network-observability.adoc
+
+:_content-type: CONCEPT
+[id="network-observability-architecture_{context}"]
+= Network Observablity Operator architecture
+
+image::network-observability-architecture.png[Network Observability eBPF export architecture]
+
+The Network Observability Operator provides the `FlowCollector` API, which is instantiated at installation and configured to reconcile the `eBPF agent`, the `flowlogs-pipeline`, and the `netobserv-plugin` components. Only a single `FlowCollector` per cluster is supported.
+
+The `eBPF agent` runs on each cluster node with some privileges to collect network flows. The `flowlogs-pipeline` receives the network flows data and enriches it with Kubernetes identifiers. If you are using Loki, the `flowlogs-pipeline` sends flowlogs data to Loki for storing and indexing. The `netobserv-plugin`, which is a dynamic {product-title} web console plugin, queries Loki to fetch network flows data. Cluster-admins can view the the data in the web console.
```

Comment:
Avoind using _it_ to avoid confusion; also, is flow logs 2 to words? I see two references in our docs and it is two.
```suggestion
The `eBPF agent` runs on each cluster node with some privileges to collect network flows. The `flowlogs-pipeline` receives the network flows data and enriches the data (?) with Kubernetes identifiers. If you are using Loki, the `flowlogs-pipeline` sends flow logs data to Loki for storing and indexing. The `netobserv-plugin`, which is a dynamic {product-title} web console plugin, queries Loki to fetch network flows data. Cluster-admins can view the the data in the web console.
```